test/shaping/run-tests.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import sys, os, subprocess
+
+def cmd(command):
+	p = subprocess.Popen (
+		command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
+	p.wait ()
+	return p.stdout.read ().decode ('utf-8'), p.returncode
+
+srcdir = os.environ.get ("srcdir", '.')
+builddir = os.environ.get ('builddir', '.')
+top_builddir = os.environ.get ('top_builddir', '../..')
```

Comment:
Maybe you should use os.path.join() instead of using the literal forward slash, for better windows compatibility

Reply:
Done